lib/presentation/home_page/components/main_page_weather_content.dart

```diff
@@ -132,10 +134,30 @@ class _MainPageWeatherContentState extends State<MainPageWeatherContent> {
                       ),
                     );
                   } else {
-                    return Center(
-                      child: Text('ładowanie'),
+                    child = Center(
+                      child: Lottie.asset(
+                        'lib/assets/lottie/61302-weather-icon.json',
+                        key: const ValueKey(0),
+                      ),
                     );
                   }
+                  return AnimatedSwitcher(
+                    // switchInCurve: Curves.fastOutSlowIn,
+                    // switchOutCurve: Curves.bounceIn,
+
+                    transitionBuilder: (child, animation) => FadeTransition(
+                      opacity: Tween<double>(begin: 0.0, end: 3.0).animate(
```

Comment:
I think opacity needs only <0,1> values.

Reply:
Ok, so i set opacity in <0,1> values and it's works :)